telegram/utils/helpers.py

```diff
@@ -55,6 +56,79 @@ def get_signal_name(signum: int) -> str:
     return _signames[signum]
 
 
+def is_local_file(obj: Optional[Union[str, Path]], absolute: bool = True) -> bool:
+    """
+    Checks if a given string is a file on local system.
+
+    Args:
+        obj (:obj:`str`): The string to check.
+        absolute (:obj:`bool`): Optional. Whether to allow only absolute paths. Defaults to
+            :obj:`True`.
+    """
+    if obj is None:
+        return False
+
+    path = Path(obj) if isinstance(obj, str) else obj
+    try:
```

Comment:
Why do we need try except here?

Reply:
`Path('https://some-domain.org').is_file()` throws an error instead of returning `False` …

Reply:
nah, `path.is_file()` can still return `False` if the passed string looks like a valid path but the file doesn't exist